_test/export.response.test.php

```diff
@@ -149,9 +149,9 @@ public function test_elements_ordernumber_and_category_are_set_according_to_page
     public function parameterProviderForNamespaceDepthTesting()
     {
         return [
-            'one' => ['home', 'home'],
-            'two' => ['home:home', 'home_home'],
-            'three' => ['home:home:home', 'home_home_home']
+            'one' => ['home', 'Home'],
```

Comment:
Nice test coverage! 1, 2, many.

Underscores are replaced with spaces. I see no test case for this. Unless I am missing something please add test cases for this; probably also mixed with colons replaced by underscore.